### PR TITLE
Updates for Firefox Accounts & Tiles

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: conf/runner.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ runs:
         - oauth.accounts.firefox.com
         - profile.accounts.firefox.com
         - verifier.accounts.firefox.com
-        - accounts.cdn.mozilla.net
```
Comment (jvehent) [inline conf/runner.yaml]: This still resolves to cloudfront. Is it no longer in use, and if so, can we get rid of the record in route53?

Comment (jbuck) [reply conf/runner.yaml]: Yep! I just deleted the CDN, S3 bucket, and route 53 record

**Comment by floatingatoll** [comment]
Note that it's not fully deleted yet, in case that's a concern:

accounts.cdn.mozilla.net. 35    IN    CNAME    d2azfwpwj8hugi.cloudfront.net
.
mozilla.net.        62732    IN    NS    ns5-65.akam.net.

**Comment by jbuck** [comment]
Haaaaah it's totally in Mozilla DNS too. Well that was a misleading DNS entry I deleted in Route 53!

**Comment by jbuck** [comment]
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1418446 to delete it within Mozilla's DNS. Thanks for the catch!

**Pull request merged**